app/assets/stylesheets/components/page_specific/_home_top.scss

```diff
@@ -0,0 +1,193 @@
+// Banner at the top of the home page
+//
+// Styleguide Home Top
+
+.home-top {
+  background: image_url('illustrated_people.png') 0 100% no-repeat;
+  background-size: 135%;
+  margin: 0;
+  position: relative;
+  width: 100%;
+  display: block;
+  padding-bottom: 50%;
+
+  @include respond-to($mq-m) {
+    background-position: 165% 108%;
+    background-size: 70%;
+    padding-bottom: 0;
+  }
+
+  @include respond-to($mq-l) {
+    background-size: 65%;
+    background-position: 141% 100%;
+
+    @if ($responsive == false) {
+      width: $mq-l;
+      margin: 0 auto;
+      float: none;
+    }
+  }
+
+  @include respond-to($mq-xl) {
+    background: none;
+    overflow: hidden;
+  }
+}
+
+.home-top-trust {
+  position: relative;
+
+  @include respond-to($mq-xl) {
+    position: absolute;
+    top: 0;
+    width: 1000px;
+  }
+}
+
+.home-top__image {
+  @include respond-to($mq-xl) {
+    width: 1350px;
+    height: 309px;
+    margin: 0 auto;
+    position: relative;
+    background: image_url('illustrated_people.png') 100% 52% no-repeat;
+    background-size: 49%;
+
+    .theme-cy & {
+      height: 351px;
+      background-position: 100% 100%;
+      background-size: 51%;
+    }
+  }
+}
+
+.home-top-trust__heading {
+  @extend %font-heading-heavy;
+  @include body(36, 42);
+  color: $color-heading-extra-small;
+  padding-top: $baseline-unit*5;
+  padding-bottom: $baseline-unit*5;
+  letter-spacing: -1.5px;
+  margin: 0;
+
+  @include respond-to(px(600), px(719)) {
```

Comment:
As discussed can we turn `px(600)` into a variable with an explanatory comment about it overlapping the image?